README.md

```diff
@@ -1,27 +1,40 @@
-# Project Match
-
+![Image alt text](https://i.imgur.com/qD4SvyR.png "Main Page")
+
+## Introduction
+Project Match is a web app created with the MERN stack with typescript integration. It's primary function is to help programmers and designers match with projects they would be interested in. Project Match will then provide team collaboration tools to support new teams with features such as a dedicated platform for mockup reviews/redlines and group chat.
+
+## Contributors
+* [Francesca Sadikin (PM and Designer)](https://github.com/serpient)
+* [luoto](https://github.com/luoto)
+* [ram](https://github.com/ilvcs)
+* [Eric Miller](https://github.com/ericmiller777)
+
+## Setting up the project
+```
+git clone https://github.com/chingu-voyage4/Bears-Team-11.git
+git remote add upstream https://github.com/chingu-voyage4/Bears-Team-11.git
+npm install -g create-react-app
+npm install
+cd bears-team-11/
+npm start
+```
 ## Running the Server
-`npm run start-server` runs the express server on port `8080`.
+- `npm run start-server` runs the express server on port `8080`.
+- Open Postman app or use your browser to reach `http://localhost:8080/`
```

Comment:
Lets provide a link to Postman for convenience: [Postman](https://www.getpostman.com/).